### DISCUSSION TITLE
how to disable automatic creation of links?

### DISCUSSION THREAD

**Discussion by WilliamAlba** [vincode-io/Zavala#267] [Support] [answered]
Hello, I am just getting started with Zavala, which is beautiful in its cross-platform simplicity. 

As I create my first outline on macOS, the app creates links automatically. How do I disable that behavior?

Example 1: My text contains "foobar.com". The URL link gets automatically created, which is kind of cool but not always desirable. I am able to delete the link by moving into the text, editing the link (cmd-K), and erasing the URL.

Example 2: My text contains "2024 Camino de Santiago". This is being interpreted as an address, which is not at all what I intend. When I attempt to delete the link by editing it as above, the link gets recreated. (This might be a setting in Apple Maps or in Zavala or somewhere else, I admit I don't know.)

Thank you!
William

**Comment by vincode-io** [comment, marked as answer]
You can't currently disable the automatic data detection and linking. I can see why you would want disable it, so I created Issue #268 to track this for a future release. I'll try to get it into the upcoming 3.1 release, but no promises.

**Reply by WilliamAlba** [reply]
Thanks very much!
